extensions/grpc/cli/src/main/java/io/quarkus/grpc/cli/GrpcCommand.java

```diff
@@ -0,0 +1,27 @@
+package io.quarkus.grpc.cli;
+
+import java.util.concurrent.Callable;
+
+import io.quarkus.picocli.runtime.annotations.TopCommand;
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Model.CommandSpec;
+import picocli.CommandLine.Spec;
+
+@TopCommand
+@Command(name = "grpc", sortOptions = false, header = "grpc CLI", subcommands = {
+        ListCommand.class, DescribeCommand.class, InvokeCommand.class })
+public class GrpcCommand implements Callable<Integer> {
+
+    @Spec
+    protected CommandSpec spec;
+
+    @CommandLine.Option(names = { "-h", "--help" }, usageHelp = true, description = "Display this help message.")
```

Comment:
Isn't this generated by picocli automatically?

Reply:
Hmmm, no idea -- copied this from one of the existing CLI commands ...
@iocanel ^ ?

Reply:
@iocanel can this be dropped? ^

Reply:
I think that it can be possibly dropped.